pages/testing.js

```diff
@@ -0,0 +1,201 @@
+          employment services. All of this is made possible by individual donations and corporate
+          partnerships.
+        </p>,
+      ]}
+    />
+
+    <Content
+      theme="gray"
+      title="Core Values"
+      hasTitleUnderline
+      columns={[
+        <ValueCard
+          name="Integrity"
+          description="We hold ourselves accountable for obtaining results
+            that fulfill our Mission and work towards our Vision.
+            As leaders, we are responsible for what we do, or fail to do.
+            We act consistently with Operation Code's mission, being honest
+            in what we do and say, and accept responsibility for our collective
+            and individual actions."
+        />,
+        <ValueCard
+          name="Transparency"
+          description="By being transparent in everything Operation Code does,
+            from our open source code base to our financial operations, our
+            community can trust that we will do our best with the resources we're given."
+        />,
+        <ValueCard
+          name="Agility"
+          description="We run fast and we run lean. When our operating environment changes,
+            and new challenges emerge, we're ready to respond. We communicate as early and as
+            often as possible, and default to overcommunication in all of our interactions.
+            We always look for a better, more effective, and more efficient way to run our
+            operations."
+        />,
+        <ValueCard
+          name="Curiosity"
+          description="Innovation is central to our mindset. We maintain an attitude of
+            continuous improvement, and we constantly look for new and better ways to
+            serve our community - we owe them nothing less."
+        />,
+        <ValueCard
+          name="Responsibility"
+          description="We exist to meet the needs of the military community.
+            We will humbly remember that their service was of the highest order and
+            that Operation Code works for them. When it comes to the welfare and
+            well-being of our community, the buck stops here."
+        />,
+        <ValueCard
+          name="Clarity"
+          description="In the words of the US Army Signal Corps, we get the
+            message through. We ensure that we are clear and concise in how we
+            communicate, in support of our operations. We are considerate of the
+            sender and the receiver, and communicate in the manner that best suits
+            the needs of the mission and the vision."
+        />,
+        <ValueCard
+          name="Community"
+          description="We look out for the people to our left and right,
+            and always keep a hand free to help. We listen to our community
+            and our people to ensure we are working towards the mission and the vision."
+        />,
+      ]}
+    />
+  </>
+);
```

Comment:
I've created testing.js page just to test whether my code is working or not in precommit hook. Will remove this page after review is done.